cpp/tests/io/parquet_common.cpp

```diff
@@ -139,8 +139,8 @@ std::unique_ptr<cudf::column> make_parquet_list_list_col(
 
   // child values
   std::vector<T> child_values(num_rows * lists_per_row * list_size);
-  T first_child_value_index = skip_rows * lists_per_row * list_size;
-  int child_value_count     = 0;
+  auto first_child_value_index = skip_rows * lists_per_row * list_size;
```

Comment:
These changes are also in another recent still under review PR but are needed here to build correctly. The other PR can take it from `branch-25.10` when this one merges.